src/Validator/Constraints/InlineConstraint.php

```diff
@@ -87,14 +133,6 @@ public function getTargets(): string
         return self::CLASS_CONSTRAINT;
     }
 
-    public function getRequiredOptions(): array
```

Comment:
this is deprecated and not working properly anymore going forward. We check and throw the exception in the constructor instead